docs/rbac.md

```diff
@@ -224,11 +224,11 @@ rolebinding "tiller-binding" created
 
 ## Helm and Role-based Access Control
 
-When running a helm client in a pod, in order for the helm client to talk to a tiller instance, it will need certain privileges to be granted. Specifically, the helm client will need to be able to create pods, forward ports and be able to list pods in the namespace where tiller is running (so it can find tiller).
+When running a helm client in a pod, in order for the helm client to talk to a Tiller instance, it will need certain privileges to be granted. Specifically, the helm client will need to be able to create pods, forward ports and be able to list pods in the namespace where Tiller is running (so it can find Tiller).
```

Comment:
While we are fixing capitalization, can you capitalize "Helm" where it occurs in this sentence?